src/Modules/Shared/Microsoft.PowerShell.Utility/Microsoft.PowerShell.Utility.psm1

```diff
@@ -737,4 +737,18 @@ function ConvertFrom-SddlString
             RawDescriptor = $rawSecurityDescriptor
         }
     }
-}
\ No newline at end of file
+}
+
+function Get-Uptime
+{
+    [CmdletBinding()]
+    [Alias("uptime")]

```

Comment:
@oising I don't like "low priority for alias", because it may cause unpredictable behavior 😕